content/en/docs/armory-admin/prometheus-monitoring.md

```diff
@@ -54,92 +59,170 @@ Access the Prometheus web interface by using the `kubectl port-forward` command.
+        enabled: true
+        version: 1.1.3
+        # This is the basic configuration for prometheus to be enabled
+        config.metrics:
           prometheus:
             enabled: true
-            add_source_metalabels: true          
+    repositories:
+      armory-observability-plugin-releases:
+        url: https://raw.githubusercontent.com/armory-plugins/armory-observability-plugin-releases/master/repositories.json
 ```
 
-Wait for all of the Spinnaker pods to be ready before proceeding to the next step. You can check the status by running the `kubectl get pods` command.  Because you are adding a sidecar to each pod, you may need to ensure you have enough capacity in your Kubernetes cluster to be able to support the additional resource requirements.
+More options for management endpoints and the plugin are available on the [Plugin readme](https://github.com/armory-plugins/armory-observability-plugin).
 
-##  Configure Prometheus to monitor Spinnaker
+### Add the ServiceMonitor
 
-There are two steps to configure Prometheus to monitor Spinnaker:
-
-- Add permissions for Prometheus to talk to the Spinnaker namespace
-- Configure Prometheus to find the Spinnaker endpoints
+Prometheus Operator uses a "ServiceMonitor" to add targets that get scraped for monitoring. The following
+example config shows how to monitor pods that are using the Observability Plugin to expose the `aop-prometheus` endpoint. Note that the example contains both the exclusion of certain services (such as Redis) and changes to the Gate endpoint to show you different options.  
 
-Add permissions for Prometheus by applying the following configuration to your cluster:
+These are examples of potential configurations. Use them as a starting point. Armory recommends that you understand how 
+they operate and find services. Adapt them to your environment.
 
 ```yaml
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: prometheus-k8s
-  namespace: spinnaker-system
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - endpoints
-  - pods
-  verbs:
-  - get
-  - list
-  - watch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
 metadata:
-  name: prometheus-k8s
-  namespace: spinnaker-system
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: prometheus-k8s
-subjects:
-- kind: ServiceAccount
-  name: prometheus-k8s
-  namespace: monitoring
+  labels:
+    app: spin
+    release: prometheus-operator
+  name: spinnaker-all-metrics
+  namespace: spinnaker
+spec:
+  endpoints:
+  - interval: 10s
+    path: /aop-prometheus
+  selector:
+    matchExpressions:
+    - key: cluster
+      operator: NotIn
+      values:
+      - spin-gate
+      - spin-gate-api
+      - spin-gate-custom
+      - spin-deck
+      - spin-deck-custom
+      - spin-redis
+      - spin-terraformer
+      - spin-dinghy
+    matchLabels:
+      app: spin
+    namespaceSelector:
```

Comment:
namespaceSelector does not exists in my prometheus versions, I just removed this part and it worked.

Reply:
👍 removing it